packages/space-opera/src/components/animation_controls/animation_controls.ts

```diff
@@ -42,10 +37,10 @@ interface AnimationControlsInterface {
 export class AnimationControls extends ConnectedLitElement {
   @query('me-checkbox#animation-autoplay') autoplayCheckbox?: CheckboxElement;
   @internalProperty() animationNames: string[] = [];
-  @internalProperty() config: AnimationControlsInterface = {};
+  @internalProperty() config: ModelViewerConfig = {};
 
   stateChanged(state: State) {
-    this.animationNames = getAnimationNames(state);
+    this.animationNames = getModelViewer()?.availableAnimations ?? [];
```

Comment:
So, regarding what else I am going do to, there's actually a lot. I'm going to remove editing-adapter completely, along with all state related to the glTF. My basic thought is that the snippet state lives in Redux and in combination with the glTF model we push over the wire is enough to reconstitute everything for the mobile view. I'm going to throw out any hope for doing incremental undos to material edits and just let model-viewer handle the state for all that itself. The material state will be serialized by `exportScene()` rather than Redux. I'm going to write a little design doc for this.

Coming back to your question above, remove animation will be another glTF edit, so possible, but not tracked by Redux or directly undoable (I'm going to stick with the Revert feature we have now instead).

Reply:
Gotcha, will you be keeping the "original" materials in redux to do an overall revert back to?

Reply:
I'll believe they're already buried in MV somewhere, but I may need to add a hook to access them. They're read-only (for a given model-load), so probably no need to include them in the state.